tgui/packages/tgui/interfaces/Keybinds/KeybindList.tsx

```diff
@@ -0,0 +1,56 @@
+/**
+ * @file
+ * @copyright 2024
+ * @author Garash2k (https://github.com/garash2k)
+ * @license MIT
+ */
+import { useState } from 'react';
+import { LabeledList, NoticeBox } from 'tgui-core/components';
+
+import { useBackend } from '../../backend';
+import { Keybind } from './Keybind';
+import { KeybindData, KeybindsData } from './types';
+
+export const KeybindList = () => {
+  const { data } = useBackend<KeybindsData>();
+  const { keys } = data;
+
+  const [focusedKey, setFocusedKey] = useState('');
+
+  return (
+    <>
+      <NoticeBox info>
+        You can only rebind keys you have access to when opening the window.
+        <br />
+        Ex: You can only change human hotkeys if you are currently human.
+      </NoticeBox>
+      <LabeledList>
+        <LabeledList.Item label="Action">
+          Corresponding Keybind
+        </LabeledList.Item>
+        {keys.sort(sortKeys).map((k) => (
```

Comment:
Find with there being a sort for alphabetizing, trying to get my head around what actions might have an id of numbers to warrant use of the custom sort function. Could you give some examples?

Reply:
Yup, example:

- Top of the list should be Up with id:2
- Down:4, Left:8, Right:16
- Then there's stuff like Run:32, Point:128 and Pull:512
- And then the rest of the IDs should be alphabetical: Say:say, OOC:ooc, Grab Intent:grab, Target Left Leg:l_leg.

If we were to sort them alphabetically, Point would be put above Up, and Run would be put between Up and Down. We can't sort the numerically because most of them are strings.